src/styles/main/card.scss

```diff
@@ -0,0 +1,114 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.main {
+  margin-top: 50px;
+}
+
+.card {
+  box-sizing: border-box;
+  position: relative;
+  background-color: #fff;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  margin-bottom: 46px;
+}
+
+.container {
+  position: relative;
+  display: grid;
+  grid-template-columns: 248px 248px 248px 248px;
+  grid-template-rows: 1fr 1fr;
+  justify-content: center;
+  justify-items: center;
+}
+
+.content-wrapper {
+  margin: 36px 16px;
+  margin-bottom: 0;
+  height: 148px;
+}
+
+.image {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.product-name {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.product-code {
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 14px;
+}
+
+.content-price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+}
+
+.price-text {
+  font-size: 12px;
+  font-weight: 400;
+  color: #616070;
+  line-height: 18px;
+}
+
+.prise-num {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.button-link {
+  text-decoration: none;
+}
+
+.button {
+  box-sizing: border-box;
+  padding: 0;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  margin-top: 16px;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
+  border: none;
+}
+
+.button:hover {
+  border: 1px solid #00acdc;
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.content-revie {
```

Comment:
There is a typo in the class name 'content-revie'. It should be 'content-review' to maintain consistency and clarity.